mitxgraders/formulagrader.py

```diff
@@ -65,54 +61,50 @@ def validate_forbidden_strings_not_used(expr, forbidden_strings, forbidden_msg,
     >>> validate_forbidden_strings_not_used(
     ... '2*sin(x)*cos(x)',
     ... ['*x', '+ x', '- x'],
-    ... 'A forbidden string was used!',
-    ... True
+    ... 'A forbidden string was used!'
     ... )
     True
 
     Fails validation if any forbidden string is used:
     >>> validate_forbidden_strings_not_used(
     ... 'sin(x+x)',
     ... ['*x', '+ x', '- x'],
-    ... 'A forbidden string was used!',
-    ... True
+    ... 'A forbidden string was used!'
     ... )
     Traceback (most recent call last):
     InvalidInput: A forbidden string was used!
     """
-    stripped_expr = casify(expr, case_sensitive).replace(' ', '')
+    stripped_expr = expr.replace(' ', '')
     for forbidden in forbidden_strings:
-        check_for = casify(forbidden, case_sensitive).replace(' ', '')
+        check_for = forbidden.replace(' ', '')
         if check_for in stripped_expr:
             # Don't give away the specific string that is being checked for!
             raise InvalidInput(forbidden_msg)
     return True
 
-def validate_only_permitted_functions_used(used_funcs, permitted_functions, case_sensitive):
+def validate_only_permitted_functions_used(used_funcs, permitted_functions):
     """
     Check that the used_funcs contains only permitted_functions
     Arguments:
         used_functions ({str}): set of used function names
         permitted_functions ({str}): set of permitted function names
-        case_sensitive (bool): whether comparison is case sensitive
     Usage
     =====
-    Case-sensitive comparison:
     >>> validate_only_permitted_functions_used(
     ... set(['f', 'sin']),
-    ... set(['f', 'g', 'sin', 'cos']),
-    ... True)
+    ... set(['f', 'g', 'sin', 'cos'])
+    ... )
     True
     >>> validate_only_permitted_functions_used(
     ... set(['f', 'Sin', 'h']),
-    ... set(['f', 'g', 'sin', 'cos']),
-    ... True)
+    ... set(['f', 'g', 'sin', 'cos'])
+    ... )
     Traceback (most recent call last):
     InvalidInput: Invalid Input: function(s) 'h', 'Sin' not permitted in answer
     """
-    casified_permitted = {casify(f, case_sensitive) for f in permitted_functions}
+    casified_permitted = {f for f in permitted_functions}
     used_not_permitted = [f for f in used_funcs
```

Comment:
Let's get rid of `casified_permitted` entirely since it's not casified anymore, and just use

```
used_not_permitted = [f for f in used_funcs
                          if f not in permitted_functions]
```

Reply:
Yup, missed that entirely. I was searching for "case" and "casify" ;-)